Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/Definitions/Enums.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Frends.AzureBlobStorage.ListBlobsInContainer.Definitions
+{
+    /// <summary>
+    ///     Authentication options
```

Comment:
I do not think there is need for indentation in this place, even though it is XML.